<!DOCTYPE html><html lang='en-GB' xml:lang='en-GB'><head><meta charset='utf-8'/><meta name='viewport' content='width=device-width, initial-scale=1.0'/><meta name='google' content='notranslate'/><meta http-equiv='Content-Language' content='en'/><link rel='icon' href='../favicon.ico' type='image/x-icon'/><title>Details IGLJ1 | Stitch</title><style>@font-face {
  font-family: Roboto;
  src: url(../Roboto-Regular.ttf);
  font-weight: normal;
}
@font-face {
  font-family: Roboto;
  src: url(../Roboto-Medium.ttf);
  font-weight: 500;
}
@font-face {
  font-family: 'Roboto Mono';
  src: url(../RobotoMono-VariableFont.ttf);
  font-weight: 100 700;
  font-style: normal;
}</style><script>assets_folder = 'report-monoclonal_PGDM1400';</script><script src='../script.js'></script><link rel='stylesheet' href='../styles.css'/></head><body class='details' onload='Setup()'><a href='../../report-monoclonal_PGDM1400.html' class='overview-link'>Overview</a><a href='#' id='back-button' class='overview-link' style='display:none;' onclick='GoBack()'>Undefined</a><div id='IGLJ1 class='info-block template-info'><h1>Segment IGLJ1</h1><h2 >Consensus Sequence<button type='button' class='user-help'><span class='mark'>?</span><div class='content'><h3>Consensus Sequence</h3>The sequence obtained by taking the highest scoring amino acid for each position. The score is calculated as the sum of all positional scores for that amino acid on this position. Any position where no reads where mapped are filled in by the sequence from the template, otherwise the template does not have any direct effect on the scoring. Insertions into the consensus sequence are only taken into account if the total score for that insertion is higher than the &#39;default&#39; score, which is calculated by summing all positional scores of all amino acids not followed by an insertion from the previous position.</div></button></h2><p class='aside-seq'>YVFTGQGTKVDKL</p><div class='annotated-consensus-sequence'><h2 >Annotated Consensus Sequence<button type='button' class='user-help'><span class='mark'>?</span><div class='content'><h3>Annotated Consensus Sequence</h3><p>The consensus sequence aligned with the template sequence. The differences are accentuated by displaying the changes from the template sequence below. On recombined templates the displayed sequence is retrieved from the original templates used in templatematching.</p></div></button></h2><button type='button' class='user-help copy-data'><span class='mark'>Copy Data</span><div class='content'><h3>Annotated Consensus Sequence (TXT)</h3><h4>Preview</h4><pre class='example'>Loading example...</pre><i>Click on the button to copy the data to your clipboard.</i></div></button><div class='annotated'><div class='names'><span>Consensus</span><span>Template</span></div><div class='CDR3'><span class='title'>CDR3</span><span>Y</span><span>Y</span><span class='dif'>&#160;</span></div><div class='CDR3'><span>V</span><span>V</span><span class='dif'>&#160;</span></div><div class='Conserved'><span>F</span><span>F</span><span class='dif'>&#160;</span></div><div class='Conserved'><span>T</span><span>.</span><span class='dif'>T</span></div><div class='Conserved'><span>G</span><span>G</span><span class='dif'>&#160;</span></div><div class='None'><span>Q</span><span>T</span><span class='dif'>Q</span></div><div class='Conserved'><span>G</span><span>G</span><span class='dif'>&#160;</span></div><div class='None'><span>T</span><span>T</span><span class='dif'>&#160;</span></div><div class='None'><span>K</span><span>K</span><span class='dif'>&#160;</span></div><div class='None'><span>V</span><span>V</span><span class='dif'>&#160;</span></div><div class='None'><span>D</span><span>T</span><span class='dif'>D</span></div><div class='None'><span>K</span><span>V</span><span class='dif'>K</span></div><div class='None'><span>L</span><span>L</span><span class='dif'>&#160;</span></div></div><textarea class='graph-data hidden' aria-hidden='true'>Consensus  YVF.GTGTKVTVL
Template   YVFTGQGTKVDKL
Difference &#160;&#160;&#160;T&#160;Q&#160;&#160;&#160;&#160;DK&#160;</textarea></div><div class='graph'><h2 >Sequence Consensus Overview<button type='button' class='user-help'><span class='mark'>?</span><div class='content'><h3>Sequence Consensus Overview</h3><p>This displays the found diversity of amino acids for each position. The amino acids are linearly scaled to the respective score. The score is calculated as the sum of all positional scores (if present otherwise the general scores) for this position in all aligned reads. Any positions where no reads where mapped are displayed as dots &#39;.&#39;. Ambiguous positions (see next section) are underlined.</p></div></button></h2><button type='button' class='user-help copy-data'><span class='mark'>Copy Data</span><div class='content'><h3>Sequence Consensus Overview (TSV)</h3><p>A TSV file with for each position in the consensus sequence all found amino acids with its score. These sets are saved as three consecutive columns filled with first the amino acid set, its width, and the score.</p><h4>Preview</h4><pre class='example'>Loading example...</pre><i>Click on the button to copy the data to your clipboard.</i></div></button><div class='sequence-logo' style='--sequence-logo-height:35px;--sequence-logo-font-size:30px;'><div class='sequence-logo-position CDR3'><span>.</span></div><div class='sequence-logo-position CDR3'><span>.</span></div><div class='sequence-logo-position Conserved'><span style='font-size:30px;transform:scaleX(1) translateX(0%)'>F</span></div><div class='sequence-logo-position Conserved'><span style='font-size:30px;transform:scaleX(1) translateX(0%)'>T</span></div><div class='sequence-logo-position Conserved'><span style='font-size:30px;transform:scaleX(1) translateX(0%)'>G</span></div><div class='sequence-logo-position'><span style='font-size:30px;transform:scaleX(1) translateX(0%)'>Q</span></div><div class='sequence-logo-position Conserved'><span style='font-size:30px;transform:scaleX(1) translateX(0%)'>G</span></div><div class='sequence-logo-position'><span style='font-size:30px;transform:scaleX(1) translateX(0%)'>T</span></div><div class='sequence-logo-position'><span style='font-size:30px;transform:scaleX(1) translateX(0%)'>K</span></div><div class='sequence-logo-position'><span style='font-size:30px;transform:scaleX(1) translateX(0%)'>V</span></div><div class='sequence-logo-position ambiguous a10' onclick='HighlightAmbiguous("a10", 10)'><span style='font-size:30px;transform:scaleX(1) translateX(0%)'>D</span></div><div class='sequence-logo-position ambiguous a11' onclick='HighlightAmbiguous("a11", 11)'><span style='font-size:13.112652232746955px;transform:scaleX(2.2878666701065504) translateX(0%)'>V</span><span style='font-size:16.887347767253043px;transform:scaleX(1.7764778941885855) translateX(0%)'>K</span></div><div class='sequence-logo-position'><span style='font-size:12.483369691093694px;transform:scaleX(2.4031972730410773) translateX(0%)'>G</span><span style='font-size:17.516630308906304px;transform:scaleX(1.712658169462339) translateX(0%)'>L</span></div></div><textarea class='graph-data hidden' aria-hidden='true'>0
1
2	F	1	0.99998
3	T	1	2
4	G	1	1.99984
5	Q	1	1.95004
6	G	1	1.99862
7	T	1	1.98923
8	K	1	1.99953
9	V	1	1.9765000000000001
10	D	1	1.91507
11	V	1	0.77522	K	1	0.99838
12	G	1	0.46352	L	1	0.65041
</textarea></div><div class='ambiguity-overview'><h2 >Variant Graph<button type='button' class='user-help'><span class='mark'>?</span><div class='content'><h3>Variant Graph</h3>All ambiguous positions, this means positions where the highest scoring amino acid has less then 90% of the total score for that position. For each option in the ambiguous position the support for each option in the next ambiguous position is show with a bar. The thickness indicates the total value of the support. The value can be set to relative to only this position (&#39;Local&#39;) or relative to all positions (&#39;Global&#39;). Empty positions indicate that no support was found to any adjoining ambiguous positions. The bars beside the options show the total higher order support for that option, it is scaled to the global maximum.</div></button></h2><p>Determine stroke width:</p><input type='checkbox' id='ambiguity-stroke-selector'/><label for='ambiguity-stroke-selector'><span>Local</span><span>Global</span></label><div id='ambiguity-wrapper' style='--max-support:1.94215782781442'><svg viewBox='0 0 140 70' width='140px' height='70px'><rect class='node a10' onclick='HighlightAmbiguous("a10", 10)' x='8px' y='2px' width='80px' height='66px'></rect><text x='10px' y='20px' class='position'>11</text><rect x='22px' y='40px' width='5px' height='12px' class='support-bar' style='--support:1.94215782781442'>K</rect><text x='10px' y='40px' class='option'>K</text><rect x='22px' y='60px' width='5px' height='12px' class='support-bar' style='--support:0.90703134983778'>V</rect><text x='10px' y='60px' class='option'>V</text><line x1=29px y1=34px x2=88px y2=34px class='support-arrow' style='--support-global:1;--support-local:1'></line><line x1=29px y1=54px x2=88px y2=54px class='support-arrow' style='--support-global:0.4670224720400272;--support-local:0.4670224720400272'></line><rect class='node a11' onclick='HighlightAmbiguous("a11", 11)' x='88px' y='2px' width='80px' height='66px'></rect><text x='90px' y='20px' class='position'>12</text><rect x='102px' y='40px' width='5px' height='12px' class='support-bar' style='--support:1.94215782781442'>L</rect><text x='90px' y='40px' class='option'>L</text><rect x='102px' y='60px' width='5px' height='12px' class='support-bar' style='--support:0.90703134983778'>G</rect><text x='90px' y='60px' class='option'>G</text><line x1='0px' y1='24px' x2='140px' y2='24px' class='baseline'></line></svg></div><div class='higher-order-graphs'><h2 >Higher Order Variant Graph<button type='button' class='user-help'><span class='mark'>?</span><div class='content'><h3>Higher Order Variant Graph</h3>This is the graph of all reads from the selected position spanning the shown ambiguous positions. The thickness of the lines shows the total intensity of the reads that support that connection. The graphs are simplified by joining ambiguous options such that the supported path is always clear. It is made from the same data as the first order ambiguity overview above. The root option nodes are shown in yellow. The 1st order support (only forward) is shown on the left. The graphs are coloured by row to make it easier to distinguish the different variants.</div></button></h2><div class='position a10' style='--max-backward-length:1;'><p class='title'>1st order support</p><p class='title'>Higher order graph</p><div class='bar' style='width:100.000%'></div><svg viewBox='0 0 100 30' width='100px' height='30px' style='--backward-length:1;'><rect class='root' x='-4px' y='0px' width='20px' height='20px'></rect><text class='option root' x='0px' y='15px'>K</text><text class='option' x='80px' y='15px'>L</text><line class='support-arrow' x1='12px' y1='10px' x2='78px' y2='10px' style='stroke-width:∞px'></line></svg><div class='bar' style='width:46.702%'></div><svg viewBox='0 0 100 30' width='100px' height='30px' style='--backward-length:1;'><rect class='root' x='-4px' y='0px' width='20px' height='20px'></rect><text class='option root' x='0px' y='15px'>V</text><text class='option' x='80px' y='15px'>G</text><line class='support-arrow' x1='12px' y1='10px' x2='78px' y2='10px' style='stroke-width:∞px'></line></svg></div><div class='position a11' style='--max-backward-length:2;'><p class='title'>1st order support</p><p class='title'>Higher order graph</p><div class='bar' style='width:100.000%'></div><svg viewBox='0 0 100 30' width='100px' height='30px' style='--backward-length:2;'><text class='option' x='0px' y='15px'>K</text><rect class='root' x='76px' y='0px' width='20px' height='20px'></rect><text class='option root' x='80px' y='15px'>L</text><line class='support-arrow' x1='12px' y1='10px' x2='78px' y2='10px' style='stroke-width:10px'></line></svg><div class='bar' style='width:46.702%'></div><svg viewBox='0 0 100 30' width='100px' height='30px' style='--backward-length:2;'><text class='option' x='0px' y='15px'>V</text><rect class='root' x='76px' y='0px' width='20px' height='20px'></rect><text class='option root' x='80px' y='15px'>G</text><line class='support-arrow' x1='12px' y1='10px' x2='78px' y2='10px' style='stroke-width:10px'></line></svg></div></div></div><div class='doc-plot'><div class='graph'><h2>Depth of Coverage of the Consensus Sequence<button type='button' class='user-help'><span class='mark'>?</span><div class='content'><h3>Bargraph</h3><p>The depth of coverage reported is the sum of all positional scores for that position. This is reported with the same position numbering as the consensus sequence.</p></div></button></h2><button type='button' class='user-help copy-data'><span class='mark'>Copy Data</span><div class='content'><h3>Depth of Coverage of the Consensus Sequence (TSV)</h3><h4>Preview</h4><pre class='example'>Loading example...</pre><i>Click on the button to copy the data to your clipboard.</i></div></button><div class='histogram' oncontextmenu='CopyGraphData()' aria-hidden='true'><span class='y-axis'><span class='max'>10</span><span class='min'>0</span></span><span class='empty'></span><span class='b CDR3' style='height:0.00%'></span><span class='l'>0</span><span class='b CDR3' style='height:0.00%'></span><span class='l'>1</span><span class='b Conserved' style='height:10.00%'></span><span class='l'>2</span><span class='b Conserved' style='height:10.00%'></span><span class='l'>3</span><span class='b Conserved' style='height:20.00%'></span><span class='l'>4</span><span class='b' style='height:19.50%'></span><span class='l'>5</span><span class='b Conserved' style='height:19.99%'></span><span class='l'>6</span><span class='b' style='height:19.89%'></span><span class='l'>7</span><span class='b' style='height:20.00%'></span><span class='l'>8</span><span class='b' style='height:19.77%'></span><span class='l'>9</span><span class='b' style='height:19.15%'></span><span class='l'>10</span><span class='b' style='height:17.74%'></span><span class='l'>11</span><span class='b' style='height:11.14%'></span><span class='l'>12</span></div><textarea class='graph-data hidden' aria-hidden='true'>Label	Value
&quot;0&quot;	0
&quot;1&quot;	0
&quot;2&quot;	1
&quot;3&quot;	1
&quot;4&quot;	2
&quot;5&quot;	1.95
&quot;6&quot;	2
&quot;7&quot;	1.99
&quot;8&quot;	2
&quot;9&quot;	1.98
&quot;10&quot;	1.92
&quot;11&quot;	1.77
&quot;12&quot;	1.11</textarea></div></div><h2>Scores</h2><table class='wide-table'><tr><th class='small-cell' >Length<button type='button' class='user-help'><span class='mark'>?</span><div class='content'><h3>Length</h3>The length of the sequence in amino acids, excluding any &#39;X&#39;s added by &#39;GapHead&#39; or &#39;GapTail&#39;.</div></button></th><th class='small-cell' >Score<button type='button' class='user-help'><span class='mark'>?</span><div class='content'><h3>Score</h3>The total score calculated by summing all sums for all placed reads on the template.</div></button></th><th class='small-cell' >Matches<button type='button' class='user-help'><span class='mark'>?</span><div class='content'><h3>Matches</h3>The total number of placed reads on the template.</div></button></th><th class='small-cell' >Unique Score<button type='button' class='user-help'><span class='mark'>?</span><div class='content'><h3>Unique Score</h3>The total score calculated by summing all sums for all reads uniquely placed on the template.</div></button></th><th class='small-cell' >Unique Matches<button type='button' class='user-help'><span class='mark'>?</span><div class='content'><h3>Unique Matches</h3>The total number of reads uniquely placed on the template.</div></button></th></tr><tr><td class='center'>12</td><td class='center'>161</td><td class='center'>3</td><td class='center'>0</td><td class='center'>0</td></tr></table><div class='alignment'><h2 >Alignment<button type='button' class='user-help'><span class='mark'>?</span><div class='content'><h3>Alignment</h3>Here all the reads that where placed on this template are show. The template sequence is shown above in blue. The numbering is relative to the template sequence. All reads are displayed with special matches squashed or stretched to match the correct number of amino acids in the template. Swapped sets, which contain the right amino acids but not in the right order, are displayed with grey over and underline. The annotation of the template sequence is shown in the background with the same key as all other annotated positions. Uniquely placed reads are displayed in bold. Hovering over a read gives more details on its placement and scoring. Clicking on a read links to the detail page of that read.</div></button></h2><button type='button' class='user-help copy-data'><span class='mark'>Copy Data</span><div class='content'><h3>Reads Alignment (FASTA)</h3>The exported data is a FASTA file containing all reads spaced by &#39;~&#39; and gaps are indicated by &#39;.&#39;. The first sequence is the template and has as header &#39;&gt;{id} template&#39;. All other sequences are reads which have as header &#39;&gt;{id} score:{score} alignment:{alignment/CIGAR}&#39;. The alignment is needed to figure out the exact placement with stretching and squashing because there is no way to fit that in a valid fasta file.<h4>Preview</h4><pre class='example'>Loading example...</pre><i>Click on the button to copy the data to your clipboard.</i></div></button><div class='buttons'><button onclick='ToggleCDRReads()'>Only show CDR reads</button><button onclick='ToggleAlignmentComic()'>Show as comic</button></div><div class='alignment-wrapper'><div class='alignment-body' style='grid-template-columns:repeat(13, 1ch);background-image:linear-gradient(to right, var(--color-secondary-o), var(--color-secondary-o)),linear-gradient(to right, var(--color-tertiary-o), var(--color-tertiary-o)),linear-gradient(to right, var(--color-tertiary-o), var(--color-tertiary-o)),linear-gradient(to right, var(--color-primary-o), var(--color-primary-o));background-size:2ch 100%,3ch 100%,1ch 100%,1ch 100%;background-position:0ch,2ch,6ch,var(--highlight-pos);background-repeat:no-repeat;'><div class='numbering'></div><div class='template'>YVFGTGTKVTVL</div><a class='a10 a11' id='aligned-Combined_3396' href='../reads/Combined_3396.html' target='_blank' style='grid-column:2 / 14;' onmouseover='AlignmentDetails("Combined_3396")' onmouseout='AlignmentDetailsClear()'><span class='insertion'>T</span>FGQGTKVDKL</a><a class='a10 a11' id='aligned-C734_017' href='../reads/C734_017.html' target='_blank' style='grid-column:2 / 14;' onmouseover='AlignmentDetails("C734_017")' onmouseout='AlignmentDetailsClear()'><span class='insertion'>T</span>FGQGTKVDVG</a><a class='a10 a11' id='aligned-C306_004' href='../reads/C306_004.html' target='_blank' style='grid-column:4 / 14;' onmouseover='AlignmentDetails("C306_004")' onmouseout='AlignmentDetailsClear()'>GQGTKVDKL</a></div></div><div id='index-menus'><div class='alignment-details' id='alignment-details-Combined_3396'><h4>Combined_3396</h4><table><tr><td>Type</td><td>Read</td></tr><tr><td>Score</td><td>56</td></tr><tr><td>Start on Template</td><td>2</td></tr><tr><td>Length on Template</td><td>10</td></tr><tr><td>Start on Read</td><td>0</td></tr><tr><td>Length of Read</td><td>11</td></tr><tr><td>Positional Score</td><td class='doc-plot'><div class='graph'><div class='histogram' oncontextmenu='CopyGraphData()' aria-hidden='true'><span class='y-axis'><span class='max'>0</span><span class='min'>0</span></span><span class='empty'></span><span class='b' style='height:NaN'></span><span class='l'>0</span><span class='b' style='height:NaN'></span><span class='l'>1</span><span class='b' style='height:NaN'></span><span class='l'>2</span><span class='b' style='height:NaN'></span><span class='l'>3</span><span class='b' style='height:NaN'></span><span class='l'>4</span><span class='b' style='height:NaN'></span><span class='l'>5</span><span class='b' style='height:NaN'></span><span class='l'>6</span><span class='b' style='height:NaN'></span><span class='l'>7</span><span class='b' style='height:NaN'></span><span class='l'>8</span><span class='b' style='height:NaN'></span><span class='l'>9</span><span class='b' style='height:NaN'></span><span class='l'>10</span></div></div></td></tr><tr><td>Alignment graphic</td><td class='sequence-match-graphic'><span class='gap-in-template' style='flex-grow:1'></span><span class='match' style='flex-grow:1'></span><span class='match' style='flex-grow:1'></span><span class='match' style='flex-grow:1'></span><span class='match' style='flex-grow:1'></span><span class='match' style='flex-grow:1'></span><span class='match' style='flex-grow:1'></span><span class='match' style='flex-grow:1'></span><span class='match' style='flex-grow:1'></span><span class='match' style='flex-grow:1'></span><span class='match' style='flex-grow:1'></span></td></tr></table></div><div class='alignment-details' id='alignment-details-C306_004'><h4>C306_004</h4><table><tr><td>Type</td><td>Read</td></tr><tr><td>Score</td><td>49</td></tr><tr><td>Start on Template</td><td>3</td></tr><tr><td>Length on Template</td><td>9</td></tr><tr><td>Start on Read</td><td>0</td></tr><tr><td>Length of Read</td><td>9</td></tr><tr><td>Positional Score</td><td class='doc-plot'><div class='graph'><div class='histogram' oncontextmenu='CopyGraphData()' aria-hidden='true'><span class='y-axis'><span class='max'>2</span><span class='min'>0</span></span><span class='empty'></span><span class='b' style='height:50.00%'></span><span class='l'>0</span><span class='b' style='height:48.26%'></span><span class='l'>1</span><span class='b' style='height:49.94%'></span><span class='l'>2</span><span class='b' style='height:49.79%'></span><span class='l'>3</span><span class='b' style='height:49.98%'></span><span class='l'>4</span><span class='b' style='height:50.00%'></span><span class='l'>5</span><span class='b' style='height:49.25%'></span><span class='l'>6</span><span class='b' style='height:49.92%'></span><span class='l'>7</span><span class='b' style='height:32.52%'></span><span class='l'>8</span></div></div></td></tr><tr><td>Alignment graphic</td><td class='sequence-match-graphic'><span class='match' style='flex-grow:1'></span><span class='match' style='flex-grow:1'></span><span class='match' style='flex-grow:1'></span><span class='match' style='flex-grow:1'></span><span class='match' style='flex-grow:1'></span><span class='match' style='flex-grow:1'></span><span class='match' style='flex-grow:1'></span><span class='match' style='flex-grow:1'></span><span class='match' style='flex-grow:1'></span></td></tr></table></div><div class='alignment-details' id='alignment-details-C734_017'><h4>C734_017</h4><table><tr><td>Type</td><td>Read</td></tr><tr><td>Score</td><td>56</td></tr><tr><td>Start on Template</td><td>2</td></tr><tr><td>Length on Template</td><td>10</td></tr><tr><td>Start on Read</td><td>0</td></tr><tr><td>Length of Read</td><td>11</td></tr><tr><td>Positional Score</td><td class='doc-plot'><div class='graph'><div class='histogram' oncontextmenu='CopyGraphData()' aria-hidden='true'><span class='y-axis'><span class='max'>2</span><span class='min'>0</span></span><span class='empty'></span><span class='b' style='height:50.00%'></span><span class='l'>0</span><span class='b' style='height:50.00%'></span><span class='l'>1</span><span class='b' style='height:49.99%'></span><span class='l'>2</span><span class='b' style='height:49.24%'></span><span class='l'>3</span><span class='b' style='height:49.99%'></span><span class='l'>4</span><span class='b' style='height:49.67%'></span><span class='l'>5</span><span class='b' style='height:50.00%'></span><span class='l'>6</span><span class='b' style='height:48.83%'></span><span class='l'>7</span><span class='b' style='height:46.50%'></span><span class='l'>8</span><span class='b' style='height:38.76%'></span><span class='l'>9</span><span class='b' style='height:23.18%'></span><span class='l'>10</span></div></div></td></tr><tr><td>Alignment graphic</td><td class='sequence-match-graphic'><span class='gap-in-template' style='flex-grow:1'></span><span class='match' style='flex-grow:1'></span><span class='match' style='flex-grow:1'></span><span class='match' style='flex-grow:1'></span><span class='match' style='flex-grow:1'></span><span class='match' style='flex-grow:1'></span><span class='match' style='flex-grow:1'></span><span class='match' style='flex-grow:1'></span><span class='match' style='flex-grow:1'></span><span class='match' style='flex-grow:1'></span><span class='match' style='flex-grow:1'></span></td></tr></table></div></div><textarea class='graph-data hidden' aria-hidden='true'>&gt;IGLJ1 template
YVFTGQGTKVDKL
&gt;Combined_3396 score:56 alignment:1I10M unique:False
~~TFGQGTKVDKL
&gt;C734_017 score:56 alignment:1I10M unique:False
~~TFGQGTKVDVG
&gt;C306_004 score:49 alignment:9M unique:False
~~~~GQGTKVDKL
</textarea></div><h2 >Template Sequence<button type='button' class='user-help'><span class='mark'>?</span><div class='content'><h3>Template Sequence</h3>The template sequence as read from the original file (without any annotations).</div></button></h2><p class='aside-seq'>YVFGTGTKVTVL</p><h2>Meta Information from fasta</h2><h3>Identifier</h3><p>IGLJ1</p><h3>Fasta header</h3><p>IGLJ1</p><h3>Originating file</h3><p>IGLJ <a href='file:////storage/khangl/MS_benchmarking/stitch-v1.5.0-linux/templates/Homo_sapiens_IGKJ,IGLJ.fasta' target='_blank' title='SHA256 checksum: 7a3a6c0fcd8ca30baccdb01df71921984b3666f2b12d65bc472ca04920394b02'>/storage/khangl/MS_benchmarking/stitch-v1.5.0-linux/templates/Homo_sapiens_IGKJ,IGLJ.fasta</a></p></div></body></html>